### PR TITLE
Move loadbalancer tf module to ansible role for crearing lb

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Move loadbalancer tf module to ansible role for crearing lb

### CODE REVIEW

File: roles/loadbalancer/tasks/build.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    name: "{{ scenario_name }}_pool_http"
+    protocol: "{{ loadblcr_protocol }}"
+    lb_algorithm: "{{ loadblcr_pool_lb_method }}"
+    listener: "{{ listener.listener.id }}"
+  register: lb_pool
+
+- name: Create members for a pool in the load balancer
+  opentelekomcloud.cloud.lb_member:
+    state: present
+    name: "{{ scenario_name }}_member_{{ my_idx|int +1 }}"
```
Comment (outcatcher) [inline roles/loadbalancer/tasks/build.yaml:32]: There is no need for +1 index

File: roles/loadbalancer/tasks/destroy.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+- name: Destroy members for a pool in the load balancer
+  opentelekomcloud.cloud.lb_member:
+    state: absent
+    name: "{{ scenario_name }}_member_{{ item }}"
+    pool: "{{ scenario_name }}_pool_http"
+  with_sequence: start=1 end="{{ loadblcr_members_count|int  }}"
```
Comment (outcatcher) [inline roles/loadbalancer/tasks/destroy.yaml]: Why in build loop is used and here with_sequence?

File: roles/loadbalancer/defaults/main.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+scenario_name: ""
+loadbl_subnet_id: ""
+loadblcr_lb_local_ip: ""
+loadblcr_pool_lb_method: "round_robin"
+loadblcr_instances: ""
```
Comment (outcatcher) [inline roles/loadbalancer/defaults/main.yaml]: Lets make count 2 by default

File: roles/loadbalancer/defaults/main.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+loadbalancer_subnet_id: ""
+loadbalancer_lb_local_ip: ""
+loadbalancer_pool_lb_method: "round_robin"
+loadbalancer_instances: ""
+loadbalancer_protocol: "tcp"
+loadbalancer_protocol_port: 80
+loadbalancer_monitor_delay: 1
+loadbalancer_monitor_timeout: 1
+loadbalancer_monitor_max_retries: 2
+loadbalancer_members_count: ""
```
Comment (outcatcher) [inline roles/loadbalancer/defaults/main.yaml]: Isn't it equal to instance count?

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**